comp/core/agenttelemetry/impl/agenttelemetry.go

```diff
@@ -376,6 +371,11 @@ func (a *atel) run(profiles []*Profile) {
 }
 
 func (a *atel) writePayload(w http.ResponseWriter, _ *http.Request) {
+	if !a.enabled {
+		httputils.SetJSONError(w, errors.New("agent-telemetry is not enabled. Please enable agent telemetry"), 400)
```

Comment:
@iglendd I'm open to change the error message display to the users, maybe a link to the documentation would be nice